sentry/templates/_helper.tpl

```diff
@@ -191,18 +191,18 @@ Set Kafka Confluent host
 {{- if .Values.kafka.enabled -}}
     {{- template "sentry.kafka.fullname" . -}}
 {{- else -}}
-    "kafka-confluent"
+    kafka-confluent
 {{- end -}}
 {{- end -}}
 
 {{/*
 Set Kafka Confluent port
 */}}
 {{- define "sentry.kafka.port" -}}
-{{- if .Values.kafka.enabled -}}
-    {{- default "9092" .Values.kafka.service.port }}
+{{- if and (.Values.kafka.enabled) (.Values.kafka.service.port) -}}
```

Comment:
Don't we have this as a default anyway in the chart? So even if it's missing it'll use the parent value and therefore will always be present?

Reply:
Theoretically yes, but for some reason it wasn't working without it.

I'll try to apply it again before merging the PR